proto/axelar/nexus/exported/v1beta1/types.proto

```diff
@@ -88,22 +88,29 @@ enum TransferDirection {
       [ (gogoproto.enumvalue_customname) = "Outgoing" ];
 }
 
+message MessageID {
+  option (gogoproto.goproto_stringer) = false;
+  string chain = 1 [ (gogoproto.casttype) = "ChainName" ];
+  string id = 2 [ (gogoproto.customname) = "ID" ];
+}
+
 message GeneralMessage {
   enum Status {
     option (gogoproto.goproto_enum_prefix) = false;
     option (gogoproto.goproto_enum_stringer) = true;
 
     STATUS_UNSPECIFIED = 0 [ (gogoproto.enumvalue_customname) = "NonExistent" ];
     STATUS_APPROVED = 1 [ (gogoproto.enumvalue_customname) = "Approved" ];
-    STATUS_EXECUTED = 2 [ (gogoproto.enumvalue_customname) = "Executed" ];
+    STATUS_SENT = 2 [ (gogoproto.enumvalue_customname) = "Sent" ];
+    STATUS_EXECUTED = 3 [ (gogoproto.enumvalue_customname) = "Executed" ];
+    STATUS_FAILED = 4 [ (gogoproto.enumvalue_customname) = "Failed" ];
   }
-
-  string id = 1 [ (gogoproto.customname) = "ID" ];
+  MessageID id = 1
+      [ (gogoproto.customname) = "ID", (gogoproto.nullable) = false ];
   string source_chain = 2 [ (gogoproto.casttype) = "ChainName" ];
   string sender = 3;
-  string destination_chain = 4 [ (gogoproto.casttype) = "ChainName" ];
-  string receiver = 5;
-  bytes payload_hash = 6;
-  Status status = 7;
-  cosmos.base.v1beta1.Coin asset = 8;
+  string receiver = 4;
+  bytes payload_hash = 5;
+  Status status = 6;
```

Comment:
should add genesis import/export with the release too, so we don't forget about it later

Reply:
ya I have a pr working on that, will do after the call contract pr in